dev/app/src/index.js

```diff
@@ -275,6 +281,19 @@ const App = {
       $("#release-count").html(i[4]);
       $("#refund-count").html(i[5]);
     }
+  },
+
+  renderSellInfo: async function (productId) {
+    const {
+      getProduct,
+      productIndex
+    } = this.instance.methods;
+    var p = await getProduct(productId).call();
```

Comment:
Get product from DB, not Blockchain.
We have an API about it. You can check it in the readme.md file on master branch

Reply:
There's an issue.
Currently, blockchainId is defined as an integer thus query API can't return the right value.
If I request '?blockchainId=1', I think the API recognizes 1 as a String value.